tests/Middleware/CsrfTest.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+
+namespace Yiisoft\Yii\Web\Tests\Middleware;
+
+use Nyholm\Psr7\Factory\Psr17Factory;
+use Nyholm\Psr7\Response;
+use Nyholm\Psr7\ServerRequest;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+use Yiisoft\Router\Method;
+use Yiisoft\Security\Random;
+use Yiisoft\Security\TokenMasker;
+use Yiisoft\Yii\Web\Middleware\Csrf;
+use Yiisoft\Yii\Web\Session\SessionInterface;
+
+final class CsrfTest extends TestCase
+{
+    private const PARAM_NAME = 'csrf';
+
+    public function testProcessIsValidTokenWithRequest()
```

Comment:
```suggestion
    public function testValidTokenInBodyResultIn200()
```

Reply:
Additionally by marking test method with `@test` in phpdoc it could become `validTokenInBodyResultIn200`.